components/authz-service/server/v2/migration.go

```diff
@@ -419,133 +420,127 @@ func customPolicyFromV1(pol *storage_v1.Policy) (*storage.Policy, error) {
 
 // Basically implements "Current Resource" -> "New Resource Names (RFR)" column of
 // https://docs.google.com/spreadsheets/d/1ccaYDJdMnHBfFgmNC1n2_S1YOnMJ-OgkYd8ySbb-mS0/edit#gid=363200100
-func convertV1Resource(resource string) ([]string, error) {
```

Comment:
Drive-by: this function's return value was never more than a slice-of-one. So, I've made it a string, and fixed the fallout.